docs/serving/configuring-autoscaling.md

```diff
@@ -179,11 +181,39 @@ template.
 spec:
   template:
     metadata:
-      autoscaling.knative.dev/metric: concurrency
+      autoscaling.knative.dev/metric: cpu
       autoscaling.knative.dev/class: hpa.autoscaling.knative.dev
 ```
 
-## Additional resources
+# Implementing your own Pod Autoscaler
+
+The Pod Autoscaler custom resource allows you to implement your own autoscaler without changing anything else about the Knative Serving system.
+
+You can implement your own Pod Autoscaler if the requirements of your workload cannot be covered by the KPA or HPA, for example if you want to use a more specialized autoscaling algorithm or if you need to use a specialized set of metrics not supported by Knative out of the box.
+
+To implement your own Pod Autoscaler, you can create a reconciler that operates on your own class of Pod Autoscaler by copying a [Knative sample controller](https://github.com/knative/sample-controller) and modifying its configuration to suit your desired use case.
```

Comment:
From here down, I think this should be converted into clear and ordered steps? Then (per my comment below about "autoscaling-base-reconciler"), you can included that information within the relative step so that its clear when and where the info about the "recommended `autoscaling-base-reconciler`" applies.

Reply:
There's only really one step to this, but I've fixed the structure a bit.